package.json

```diff
@@ -1,13 +1,18 @@
 {
   "private": true,
   "scripts": {
-    "clean": "rimraf output && rimraf .pulp-cache",
+    "clean": "rimraf output && rimraf .pulp-cache && rimraf bench/output && rimraf bench/.pulp-cache",
+    "install": "bower i && cd bench && bower i",
```

Comment:
I'd call it `installation` so there is no confusion with native used of `install` ie if you try use yarn to run this it won't work. `yarn install` 😞  (The rest of the code is too magical for me but looks good 😺 )

Reply:
Or `postinstall`

Reply:
what are you doing here @toastal ? we've provided the least useful feedback 😂